src/main/java/org/inferred/freebuilder/processor/GeneratedBuilder.java

```diff
@@ -442,7 +447,10 @@ private void addValueTypeEqualsOverride(SourceBuilder code) {
   }
 
   private void addValueTypeHashCode(SourceBuilder code) {
-    FieldAccessList fields = getFields(generatorsByProperty.keySet());
+    Stream<Property> properties = generatorsByProperty.keySet()
```

Comment:
Don't assign streams to temporary variables, as it encourages using them twice, but streams can only be iterated once. Use `.collect(toList())`, or consume them at point of creation.

Reply:
Sure, I'll inline the local variables. I get weary of calling `.collect(toList())` aggressively because it can happen that someone calls `collect` to then create another stream (use the original stream) or use in for loop instead instead of `foreach`.